packages/fluentui/react-northstar/src/components/SplitButton/SplitButton.tsx

```diff
@@ -162,7 +173,7 @@ class SplitButton extends AutoControlledComponent<WithAsProp<SplitButtonProps>,
     styles,
     unhandledProps,
   }: RenderResultConfig<MenuButtonProps>): React.ReactNode {
-    const { button, disabled, menu, primary, secondary, toggleButton, size } = this.props;
+    const { button, disabled, menu, primary, secondary, toggleButton, size, position, align } = this.props;
```

Comment:
```suggestion
    const { button, disabled, menu, primary, secondary, toggleButton, size, position, align, flipBoundary, overflowBoundary, positionFixed, offset, unstable_pinned } = this.props;
```

Reply:
Let's also add a UT to `SplitButton-test`, it will catch this:

```diff
describe('SplitButton', () => {
  isConformant(SplitButton, { autoControlledProps: ['open'] });
+  implementsPopperProps(SplitButton, { requiredProps: { open: true }})
```

Reply:
Done